src/main/java/com/openlattice/edm/PostgresEdmTypeConverter.java

```diff
@@ -70,6 +70,8 @@ public static PostgresDatatype map( EdmPrimitiveTypeKind edmType ) {
     public static PostgresDatatype mapToArrayType( EdmPrimitiveTypeKind edmType ) {
         switch ( edmType ) {
             case String:
+            case Binary:
+            case GeographyPoint:
```

Comment:
It is not isolated although this particular method is unused, the above `map()` method is used in DataTable and DataQueries when determining the correct column type to set so I think changing it would require some kind of migration. I'm super down with doing that at some point though: https://jira.openlattice.com/browse/LATTICE-2572